ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -115,6 +115,65 @@ presubmits:
           name: prow-job-cluster-launch-installer-e2e
         name: job-definition
     trigger: ((?m)^/test( all| e2e-aws-all),?(\s+|$))
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/e2e-openstack
+    decorate: true
+    name: pull-ci-openshift-installer-master-e2e-openstack
+    optional: true
+    rerun_command: /test e2e-openstack
+    run_if_changed: ^([^d]|d(d|o(d|cd))*([^do]|o([^cd]|c[^ds])))*(d(d|o(d|cd))*(oc?)?)?$
+    skip_cloning: true
+    spec:
+      containers:
+      - command:
+        - ci-operator
+        - --give-pr-author-access-to-namespace=true
+        - --artifact-dir=$(ARTIFACTS)
+        - --secret-dir=/usr/local/e2e-openstack-cluster-profile
+        - --template=/usr/local/e2e-openstack
+        - --target=e2e-openstack
+        - --give-pr-author-access-to-namespace
+        env:
+        - name: CLUSTER_TYPE
+          value: openstack
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-installer-master.yaml
+              name: ci-operator-configs
+        - name: JOB_NAME_SAFE
+          value: e2e-openstack
+        - name: TEST_COMMAND
+          value: TEST_FOCUS='Secrets should be consumable from pods in volume with
+            defaultMode set' run-tests
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          limits:
+            cpu: 500m
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /usr/local/e2e-openstack-cluster-profile
+          name: cluster-profile
+        - mountPath: /usr/local/e2e-openstack
+          name: job-definition
+          subPath: cluster-launch-installer-e2e.yaml
```

Comment:
@bbguimaraes do we generate this one?

Reply:
The template, yes, but not the profile (which doesn't seem to exist).

Reply:
OK, can you help @flaper87 use the generator here

Reply:
@stevekuznetsov any points on how to use the generator? As I mentioned in the PR description, I'll need some guidance as I'm new to the community and the infrastructure. 

thanks a bunch for your reviews! :+1:

Reply:
I ran `./hack/update-generated-config.sh`